docs/vars.md

```diff
@@ -107,7 +107,8 @@ Stack](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/dns-stack.m
 * *docker_options* - Commonly used to set
   ``--insecure-registry=myregistry.mydomain:5000``
 * *docker_plugins* - This list can be used to define [Docker plugins](https://docs.docker.com/engine/extend/) to install.
-* *containerd_config* - Controls some parameters in containerd configuration file (usually /etc/containerd/config.toml).
+* *containerd_default_runtime_name* - Sets the default Containerd runtime used by the Kubernetes CRI plugin.
+* *containerd_runtimes* - Sets the Containerd runtimes used by the Kubernetes CRI plugin.
```

Comment:
Why do we need to specify containerd_default_runtime_name?
Isn't it fine to specify a single containerd_runtime only instead?

I could not find a situation we need to configure multiple runtimes and select the default from them.

Reply:
I've changed the variable to `containerd_runtime`.

If someone comes across a use case where they need to specify multiple container runtimes, then they can raise a new PR.

Reply:
I want runc+kata and containerd_default_runtime_name makes perfect sense and maps 1:1 to containerd option name. Not sure what the confusion is about.